debian/control

```diff
@@ -33,8 +33,8 @@ Architecture: any
 Depends: ${misc:Depends},
          ${shlibs:Depends},
          qml-module-qtmultimedia | qml-module-qtmultimedia-gles,
-         qtdeclarative5-ubuntu-ui-toolkit-plugin | qtdeclarative5-ubuntu-ui-toolkit-plugin-gles,
```

Comment:
I would like to keep `qml-module-ubuntu-components-gles` as this might be useful for newer ubuntu builds or debian.

Reply:
There is no such package any more. It was from an old hack where parts of Qt and Ubuntu that needed to be compiled differently for GL and GLES were built twice with different binary packages output. That is no longer the case, and the packages are just always built with GLES enabled on ARM architectures.